hoot-core/src/main/cpp/hoot/core/io/OgrUtilities.cpp

```diff
@@ -63,8 +63,6 @@ void OgrUtilities::loadDriverInfo()
   _drivers.push_back(OgrDriverInfo(".gpx",      "GPX",            true,     true,   GDAL_OF_VECTOR));
   _drivers.push_back(OgrDriverInfo(".kml",      "KML/LIBKML",     true,     true,   GDAL_OF_ALL));
   _drivers.push_back(OgrDriverInfo(".kmz",      "LIBKML",         true,     true,   GDAL_OF_ALL));
-  _drivers.push_back(OgrDriverInfo(".json",     "GeoJSON",        true,     true,   GDAL_OF_VECTOR));
-  _drivers.push_back(OgrDriverInfo(".geojson",  "GeoJSON",        true,     true,   GDAL_OF_VECTOR));
```

Comment:
Remove the support for reading/writing GeoJSON using GDAL

Reply:
+1